pages/api/category/[id]/index.ts

```diff
@@ -52,12 +52,11 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
         stories
       };
 
-      res.status(200).json(apiResp);
-      res.end();
-    } else {
-      res.status(404);
+      return res.status(200).json(apiResp);
```

Comment:
Changes like this were the key fixes to the last of the issues.

Reply:
Ah, thats a good catch. Is this something worth writing up so others don't have to feel so much pain in the future?